examples/application_commands/localization.py

```diff
@@ -0,0 +1,97 @@
+from nextcord import Interaction, Locale, SlashOption
+from nextcord.ext import commands
+
+TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
+BOT_TOKEN = ""  # Replace with your bot token
+
+
+bot = commands.Bot()
+
+
+@bot.slash_command(
+    name="hello",
+    name_localizations={
+        Locale.de: "hallo",
+        Locale.fr: "bonjour",
+        Locale.es_ES: "hola",
+    },
+    description="Description of the command in English",
+    description_localizations={
+        Locale.de: "Beschreibung des Befehls auf Deutsch",
+        Locale.fr: "Description de la commande en français",
+        Locale.es_ES: "Descripción del comando en español",
+    },
+    guild_ids=[TESTING_GUILD_ID],
+)
+async def hello_command(interaction: Interaction):
+
+    if interaction.locale == "de":
+        await interaction.response.send_message("Hallo Welt!")
+    elif interaction.locale == "fr":
+        await interaction.response.send_message("Bonjour le monde!")
+    elif interaction.locale == "es-ES":
+        await interaction.response.send_message("¡Hola Mundo!")
+    else:
+        await interaction.response.send_message("Hello, world!")
```

Comment:
These can use the Locale enum also for consistency?

```suggestion
    if interaction.locale == Locale.de:
        await interaction.response.send_message("Hallo Welt!")
    elif interaction.locale == Locale.fr:
        await interaction.response.send_message("Bonjour le monde!")
    elif interaction.locale == Locale.es_ES:
        await interaction.response.send_message("¡Hola Mundo!")
    else:
        await interaction.response.send_message("Hello, world!")
```

Reply:
This applies to both commands

Reply:
It seems like I was mistaken about this. It appears the comparison with string and Locale does not currently work the way I expected it would. Maybe in the future this will be valid, but for now the previous way you had it was correct. Sorry about that.